tools/protobench/cli_flags.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Apache License 2.0.
+ * See the file "LICENSE" for details.
+ */
```

Comment:
This file has the license header (probably pasted by accident) and the other new ones do not. Should this header be added for each new file? Not sure whether the header was only required temporarily or whether we have to add it for every new file going forward.

Reply:
In general, when we copy a file or a significant amount of code from a file with a license header, we should keep (or copy) the original license header as well. But this should be clarified by the maintainers, together with CNCF legal.

In this case, I copied and amended the code.